rules-tests/DowngradePhp70/Rector/FunctionLike/DowngradeThrowableTypeDeclarationRector/Fixture/nullable.php.inc

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Rector\Tests\DowngradePhp70\Rector\FunctionLike\DowngradeThrowableTypeDeclarationRector\Fixture;
+
+class Fixture
+{
+    public function someFunction(?\Throwable $anything): ?\Throwable
+    {
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Tests\DowngradePhp70\Rector\FunctionLike\DowngradeThrowableTypeDeclarationRector\Fixture;
+
+class Fixture
+{
+    /**
+     * @param \Throwable|null $anything
+     * @return \Throwable|null
```

Comment:
This does not preserve formatting (changes `?\Throwable` to `\Throwable|null`. But that is probably fine.

Reply:
that's actually fine https://phpstan.org/r/5c17f3ca-4b93-44e4-9597-b21bb9b784c5 , improvement maybe in separate PR.